postman/Find Subscribers Across Lists.postman_collection.json

```diff
@@ -0,0 +1,164 @@
+{
+	"info": {
+		"_postman_id": "a3f9d2bf-beea-4df2-84d7-b11576abd0a3",
+		"name": "Find Subscribers Across Lists",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "Get Account",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"id": "74e7dce1-167a-4675-8431-bdf2223fcd25",
+						"exec": [
+							"pm.test(\"Successfully get an account Id\", function(){",
+							"    pm.response.to.have.status(200);",
+							"    var jsonData = pm.response.json();",
+							"    pm.expect(jsonData.entries[0].id).to.be.a('number')",
+							"    pm.globals.set(\"account_id\",jsonData.entries[0].id )    ",
+							"});"
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"request": {
+				"method": "GET",
+				"header": [],
+				"body": {
+					"mode": "raw",
+					"raw": ""
+				},
+				"url": {
+					"raw": "{{base_URL}}/1.0/accounts",
```

Comment:
base_URL is not documented in the README. I recommend not making this a variable, like in the broadcasts example, since it's not useful to change it in the context of api examples.